lib/workspace.rb

```diff
@@ -0,0 +1,39 @@
+require_relative "channel"
+require "httparty"
+require "dotenv"
+
+Dotenv.load
+
+module SlackCLI
+  class Workspace
+    attr_reader :users, :channels
+    USER_API = "https://slack.com/api/users.list".freeze
+    CHANNEL_API = "https://slack.com/api/channels.list".freeze
+    KEY = ENV["SLACK_API_TOKEN"]
+
+    def initialize
+      @users = SlackCLI::User.list(USER_API, KEY)
+      @channels = SlackCLI::Channel.list(CHANNEL_API, KEY)
+    end
+
+    def self.users_details(user)
```

Comment:
These methods don't make sense a class methods since each Workspace has a list of users it could also keep track of the currently selected user and this method as an instance method could just return `selected.details`.